pkg/manager/manager_deep_test.go

```diff
@@ -215,7 +215,7 @@ func setUpDeepTest(t *testing.T) (*Manager, *AllMocks) {
 	assert.Assert(t, networkChange1 != nil)
 	err = mgrTest.NetworkChangesStore.Create(networkChange1)
 	assert.NilError(t, err, "Unexpected failure when creating new NetworkChange")
-	time.Sleep(time.Millisecond * 10)
+	time.Sleep(time.Millisecond * 100)
```

Comment:
to ensure proper testing could we have a watch on the store to continue the test when we receive the watch event or fail the test after a certain amount of time ?